urllib3/util/connection.py

```diff
@@ -1,14 +1,54 @@
 import socket
 try:
-    from select import poll, POLLIN
+    from select import poll, POLLIN, POLLOUT, POLLERR
 except ImportError:  # `poll` doesn't exist on OSX and other platforms
     poll = False
     try:
-        from select import select
+        from select import select as select
```

Comment:
Oh I had been mucking with naming that `_select` and the function we define `select` but then realized there was a better API for what we needed.